extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,286 @@
+  background: black !important;
+  width: min(650px, 90%) !important;
+  border-radius: 10px !important;
+  padding: 40px !important;
+  position: relative !important;
+  transition: transform 0.2s var(--i-amphtml-story-dev-tools-material-easing-accelerated), opacity 0.2s var(--i-amphtml-story-dev-tools-material-easing-accelerated) !important;
+  transform: translateY(5vh) !important;
+  opacity: 0 !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-bg[active] {
+  background: var(--i-amphtml-story-dev-tools-gray-transparent-1) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-bg[active] .i-amphtml-story-dev-tools-device-dialog-container  {
+  transform: translateY(0) !important;
+  opacity: 1 !important;
+  transition: transform 0.2s var(--i-amphtml-story-dev-tools-material-easing-decelerated), opacity 0.2s var(--i-amphtml-story-dev-tools-material-easing-decelerated) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container h1 {
+  font-size: 12px !important;
+  text-transform: uppercase !important;
+  opacity: .4 !important;
+  letter-spacing: 1px !important;
+  font-weight: normal !important;
+  margin: 0px !important;
+  margin-bottom: 5px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container h1:not(:first-child) {
+  margin-top: 25px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container.i-amphtml-story-dev-tools-device-dialog-add-devices h1 {
+  margin-inline-start: 15px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link {
+  color: var(--i-amphtml-story-dev-tools-white-transparent-3) !important;
+  text-decoration: none !important;
+  display: block !important;
+  padding: 10px 0 !important;
+  transition: color 0.2s var(--i-amphtml-story-dev-tools-material-easing-standard) !important;
+  display: flex !important;
+  align-items: center !important;
+  justify-content: space-between !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link:hover {
+  color: white !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link:not(:last-child) {
+  border-bottom: 1px solid var(--i-amphtml-story-dev-tools-white-transparent-1) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-arrow {
+  background: url('data:image/svg+xml;charset=utf-8, <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32" fill="none"><path d="M15.9998 10.6667L15.0598 11.6067L18.7798 15.3333H10.6665V16.6667H18.7798L15.0598 20.3933L15.9998 21.3333L21.3332 16L15.9998 10.6667Z" fill="white"/></svg>') !important;
+  width: 32px !important;
+  height: 32px !important;
+  opacity: 0.6 !important;
+  transition: opacity 0.2s var(--i-amphtml-story-dev-tools-material-easing-standard), transform 0.2s var(--i-amphtml-story-dev-tools-material-easing-standard) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link:hover .i-amphtml-story-dev-tools-device-dialog-arrow {
+  opacity: 1 !important;
+  transform: translateX(4px) !important;
```

Comment:
This will break the users attention from the link text. 
It may seem trivial but this animation is a distraction.

The motion principles on material IO are very good when thinking about this:
https://material.io/design/motion/understanding-motion.html#hierarchy